src/mvba/abba/message.rs

```diff
@@ -51,10 +60,19 @@ pub struct MainVoteAction {
     pub sig_share: SignatureShare,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+pub struct DecisionAction {
+    pub round: usize,
+    pub value: MainVoteValue,
+    pub justification: MainVoteJustification,
```

Comment:
DesicionAction doesn't need justification. Only threshold signature is enough.

Reply:
Oh you're right! I assumed we would need it for validating the threshold signature. Ok let me push a fix